### PR TITLE
Feature/admin video

### PR DESCRIPTION
**Pull request by rajeshdeveloper99**
1. add video section ( admin Dashboard )
2. integrate api 
3. remove classname 
4. delete api integrate

### CODE REVIEW

File: src/Components/CommonComponent/Table/Table.tsx [hunk 1]
```diff
@@ -75,6 +76,7 @@ const TableData = ({ data, columns, deleteroom, type, handleEdit }: TablePropsTy
     }
   };
 
+  console.log('sortedData', sortedData);
```
Comment (Ashok-tgc) [inline src/Components/CommonComponent/Table/Table.tsx]: remove console.log

File: src/app/spectatorDashboard/Room/page.tsx [hunk 2]
```diff
@@ -90,7 +90,7 @@ const Room = () => {
         <div className={styles.sidebar_wrapper}>
           <Navbar />
           <div className={styles.inner_specter_cls}>
-            <h1 className={styles.r_main_title}>Rooms</h1>
+            <h1 className={styles.r_main_title}>Rooms vfgjhgjjgh</h1>
```
Comment (Ashok-tgc) [inline src/app/spectatorDashboard/Room/page.tsx]: is this text required?

File: src/Components/CommonComponent/Table/Table.tsx (hunk 1 above)
Comment (rajeshdeveloper99) [reply src/Components/CommonComponent/Table/Table.tsx]: done

File: src/app/spectatorDashboard/Room/page.tsx (hunk 2 above)
Comment (rajeshdeveloper99) [reply src/app/spectatorDashboard/Room/page.tsx]: done

**Pull request merged**